skyvern/webeye/browser_factory.py

```diff
@@ -146,8 +144,21 @@ def build_browser_args() -> dict[str, Any]:
         har_dir = (
             f"{settings.HAR_PATH}/{datetime.utcnow().strftime('%Y-%m-%d')}/{BrowserContextFactory.get_subdir()}.har"
         )
+
+        preference = {
+            "plugins": {
+                "always_open_pdf_externally": True,
+                "open_pdf_in_system_reader": True,
+            },
+        }
+        user_data_dir = make_temp_directory(prefix="skyvern_browser_")
+        preference_dir = os.path.join(user_data_dir, "Default")
+        os.makedirs(preference_dir, mode=0o777, exist_ok=True)
+        with open(preference_dir + "/Preferences", "w") as f:
+            json.dump(preference, f)
+
         return {
-            "user_data_dir": make_temp_directory(prefix="skyvern_browser_"),
```

Comment:
adding a new key like `downloads_path` here is able to change the default download directory. no need to change the logic in the `set_download_file_listener` function